.github/workflows/enos-ci-bootstrap.yml

```diff
@@ -0,0 +1,26 @@
+name: enos-ci-bootstrap
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - enos/ci/**
+      - .github/workflows/enos-ci-bootstrap*
+  push:
+    branches:
+      - main
+    paths:
+      - enos/ci/**
+      - .github/workflows/enos-ci-bootstrap*
+
+jobs:
+  bootstrap-ci-oss:
+    if: ${{ github.event.repository.name == 'boundary' }}
+    uses: ./.github/workflows/enos-ci-bootstrap-oss.yml
+    secrets: inherit
+
+  bootstrap-ci-ent:
```

Comment:
I think we should move the ent job to another workflow in ent repo only.

Reply:
Done.